src/bin/pred.rs

```diff
@@ -73,6 +77,15 @@ define_language! {
   }
 }
 
+/// Return a non-zero constant.
+fn mk_constant(n: &BigInt, d: &BigInt) -> Option<Constant> {
```

Comment:
Why have this return an Option? Seems simpler to just panic if `d` is zero, since we know it is only called with `d` being `1`.

Reply:
yeah I guess we can here, in the future if we end up using it more generally, can change it back to Option.